lib/takeaway.rb

```diff
@@ -0,0 +1,56 @@
+require_relative 'menu'
+
+class Takeaway
+  attr_reader :menu, :basket, :basket_total, :price
+
+  def initialize
+    @menu = Menu.new
+    @basket = []
+  end
+
+  def show_menu
+    print menu.dishes
+  end
+
+  def add(like, count = 1)
```

Comment:
`like` is throwing me off a little, `like` is a dish in this context.

Potentially changing variable name `like` to `dish` for easier understanding.